packages/api/src/ITooltip.css

```diff
@@ -6,6 +6,7 @@
   color: #fff;
   border-radius: 2px;
   pointer-events: none !important;
+  white-space: nowrap;
```

Comment:
This is causing text to spill out of the "bubble":
![image](https://user-images.githubusercontent.com/966863/45694353-01b52e00-bb57-11e8-9d3f-f1a3aeee7ec6.png)

Reply:
I've removed the nowrap line. Its only purpose was to prevent the "north" direction from being chosen when the "east" or "west" would have looked nicer.

Reply:
I've got a version that always renders it nicely, gotta get it into PR shape though